### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by stavares843**
## Proposed changes

Fix typos across codebase.

## Types of changes

What types of changes does your code introduce to this repository?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update

### CODE REVIEW

File: docs/services/windows.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ Below is an example of an `smb.conf` for a Samba installation,
        path = /home/demo/share
        guest ok = yes
        read only = yes
-       browseable = yes
```
Comment (jayjb) [inline docs/services/windows.rst:49]: this is config setting.

File: opencanary/test/module_test.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def test_log_in_to_http_with_basic_auth(self):
         """
         request = requests.post('http://localhost/', auth=('user', 'pass'))
         # Currently the web server returns 200, but in future it should return
-        # a 403 statuse code.
+        # a 403 statutes code.
```
Comment (jayjb) [inline opencanary/test/module_test.py:126]: this should be `status`

**Review by jayjb — CHANGES_REQUESTED** [review]

File: docs/services/windows.rst [hunk 3]
```diff
@@ -46,7 +46,7 @@ Below is an example of an `smb.conf` for a Samba installation,
...
+       browsable = yes
```
Comment (stavares843) [inline docs/services/windows.rst:49]:
```suggestion
       browseable = yes
```

File: opencanary/test/module_test.py (hunk 2 above)
Comment (stavares843) [inline opencanary/test/module_test.py:126]:
```suggestion
        # a 403 status code.
```

Comment (stavares843) [reply opencanary/test/module_test.py]: updated 🔨

File: docs/services/windows.rst (hunk 1 above)
Comment (stavares843) [reply docs/services/windows.rst]: reverted 🔨

**Review by jayjb — APPROVED** [review]

**Pull request merged**